tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/polyparser/ArcHybridTransitionSystem.scala

```diff
@@ -341,7 +344,8 @@ case class ArcHybridRequestedArcInterpretation(
           PreviousLinkGretelRef(state).headOption, requestedArc.arcLabel
         ) match {
             case (Some(crumb), Some(gretel), Some(reqLabel)) =>
-              Set(crumb, gretel) == arcTokens && arcLabel != reqLabel
+              Set(crumb, gretel) == arcTokens &&
+              DependencyParsingTransitionSystem.getArcLabelSymbol(arcLabel) != reqLabel
```

Comment:
Below you indent multi-line conditional clauses but don't here (and above).  I think our `format` plugin likes it indented (like below).  Maybe just run `sbt format`?